Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/BaseClasses/Effectiveness.mo

```diff
@@ -0,0 +1,165 @@
+within Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.BaseClasses;
+model Effectiveness
+  "Model for calculating the heat exchange effectiveness of heat exchangers"
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Modelica.Units.SI.Efficiency epsS_cool_nominal(max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsL_cool_nominal(max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsS_cool_partload(max=1) = 0.75
+    "Partial load (75%) sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsL_cool_partload(max=1) = 0.75
+    "Partial load (75%) latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsS_heat_nominal(max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsL_heat_nominal(max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsS_heat_partload(max=1) = 0.75
+    "Partial load (75%) sensible heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsL_heat_partload(max=1) = 0.75
+    "Partial load (75%) latent heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.VolumeFlowRate v_flow_sup_nominal(min = 100*Modelica.Constants.eps)
```

Comment:
In our naming convention, and in all the control implementations, we call this `VSup_flow_nominal`. Please check and change variable  names for consistency.